docs/old-reference-guide/modules/messaging-concepts/pages/message-intercepting.adoc

```diff
@@ -345,6 +345,17 @@ By default, it reacts to any `Message` implementation.
 If an `EventMessage` specific interceptor is desired, the `messageType` parameter on the annotation should be set to `EventMessage.class`.
 . For even more fine-grained control of which messages should react to the interceptor, the `payloadType` contained in the `Message` to handle can be specified.
 
+[Note]
+.Intercepting `@EventSourcingHandler` annotated methods on Aggregate Members
+====
+The `@MessageHandlerInterceptor` **is not** intended to intercept events that are passed to `@EventSourcingHandler` annotated methods in aggregates.
+Although it works for the aggregate root, this is a mere side effect of the implemented handler support.
+Due to this, `@EventSourcingHandler` annotated methods on aggregate members does not work at all.
```

Comment:
Unfortunately this comment is not right.

The event is intercepted for the entity itself. This can be the aggregate root, or an aggregate member.

So let's say we have the following structure

Aggregateroot with Interceptor A and EventSourcingHandler
AggregateMember with Interceptor B and EventSourcingHandler

Now, if the aggregate needs to apply an event for the member, it will only call the Interceptor B. 
Interceptors in the class itself are honored, but the entity hierarchy is not. 

This is different from commands, where both interceptor A and B would be called.

Reply:
Ah, I didn't catch that from your ticket description. Thanks for clarifying.

I then think we should adjust the description entirely. This page does not describe at all that interceptors work in an hierarchy that adheres to aggregate members.
Hence, I think it better to explain that, and _there_ have a note that this does not apply for aggregate members. 

So, I'll go that direction. I'll nudge you once it's in place.